cmake/winml_unittests.cmake

```diff
@@ -65,6 +65,17 @@ add_winml_test(
 )
 target_precompiled_header(winml_test_api testPch.h)
 
+file(GLOB winml_test_scenario_src CONFIGURE_DEPENDS "${WINML_TEST_SRC_DIR}/scenario/cppwinrt/*.cpp")
+add_winml_test(
```

Comment:
>add_winml_test [](start = 0, length = 14)

do you need the "DEPENDS" variable here as well for the cppwinrt sdk headers? (I think you and Ori were working on that for API tests)